api/src/main/java/io/druid/data/input/InputSplit.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.data.input;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Input unit for distributed batch ingestion. Used in {@link FiniteFirehoseFactory}.
+ */
+public class InputSplit<T>
+{
+  private final T split;
+
+  @JsonCreator
+  public InputSplit(@JsonProperty("split") T split)
+  {
+    this.split = split;
+  }
+
+  @JsonProperty("split")
+  public T get()
```

Comment:
`getSplit` is more common syntax

Reply:
Yeah, but I think most of use cases would be like this:

```java
final InputSplit split = someMethodToGetSplit();
...
final T actualSplit = split.get();
```

If we rename this method, the code would be like `split.getSplit()`. Do you think this is better? I have no strong opinion here.